CONTRIBUTING.md

```diff
@@ -145,3 +145,16 @@ Before submitting a pull request, please make sure that:
 ## Contributing documentation
 
 Please format and lint documentation with `task docs`.
+
+## Contributing code on Windows
+
+In order to contribute code on Windows, please make sure that:
+
+* You have a Windows machine with `git` installed.
+* You have Go (1.18 and above) installed directly on your machine and have `go` in your PATH.
```

Comment:
If we move the whole section up, we don't have to specify the version again

Reply:
Fixed